pkg/scheduler/framework/plugins/nodeunschedulable/node_unschedulable.go

```diff
@@ -25,8 +25,7 @@ import (
 	framework "k8s.io/kubernetes/pkg/scheduler/framework/v1alpha1"
 )
 
-// NodeUnschedulable is a plugin that priorities nodes according to the node annotation
-// "scheduler.alpha.kubernetes.io/preferAvoidPods".
+// NodeUnschedulable is a plugin that checks if a pod can be scheduled on a node with Unschedulable spec.
```

Comment:
```suggestion
// NodeUnschedulable is a plugin that checks if a pod can be scheduled on a unschedulable node according to its tolerations.
```

Reply:
Hi, thanks for your suggestion, I have updated the commit